ingest/build-configs/nextstrain-automation/upload.smk

```diff
@@ -29,7 +29,7 @@ rule upload_to_s3:
         cloudfront_domain=config["cloudfront_domain"],
     shell:
         """
-        ./vendored/upload-to-s3 \
+        {params.current_basedir}/../../../shared/vendored/scripts/upload-to-s3 \
```

Comment:
Oops, latest [ingest failed](https://github.com/nextstrain/nipah/actions/runs/18169673323/job/51721011839) due to this change.  Totally missed that `current_basedir` was not added to the params above.

Reply:
Fixed in https://github.com/nextstrain/nipah/pull/18